src/Commands/UserCommands/HelpCommand.php

```diff
@@ -69,11 +69,13 @@ public function execute()
             );
 
             foreach ($command_objs as $command) {
-                $text .= sprintf(
+                if($command->getShowInHelp()) {
```

Comment:
Instead of checking for `true` and putting all the code into the `if` block, it's nicer to test for `false` and just continue with the next element.
e.g.
```php
if (!$command->showInHelp()) {
    continue;
}
```
Also, remember to keep your code nice and clean, so make the indentation correct 👍